tests/snappi/variables.py

```diff
@@ -0,0 +1,27 @@
+line_card_choice = 'chassis_multi_line_card_multi_asic'
```

Comment:
What are the meanings of line card and ASIC here? It seems to me that line card == ASIC in chassis context?

Reply:
Line card is individual slot in chassis switch. Each line card can have multiple asic.